### PR TITLE
'TableFeature'

### PR DESCRIPTION
**Pull request by bignatue**

### CODE REVIEW

File: src/Containers/TableFeature/TableFeature.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+<<<<<<< HEAD
```
Comment (ralph-dev) [inline src/Containers/TableFeature/TableFeature.stories.tsx:2]: Merge issue this code won't run, make sure you test your work!

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Merge issues all over code, attach a picture to the PR once you've tested it this will break everything

File: src/Containers/TableFeature/TableFeature.stories.tsx (hunk 1 above)
Comment (bignatue) [reply src/Containers/TableFeature/TableFeature.stories.tsx]: hi, @ralph-dev files are now updated issue has been fixed.

**Comment by ralph-dev** [comment]
@bignatue Can you revert the last commit since you changed all the files btw @sahandfarahani Can you help Brian with this

File: src/Containers/TableFeature/TableFeature.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+];
+
+export default {
+    title: createStoryTitle('TableFeature'),
+    component: TableFeature,
+    args: {
+        headingOne: 'Simplified payout process',
+        headingTwo: 'Automate payout workflows',
+        title:      'Automated payouts',
+        paragh:     'Stop uploading spreadsheets and juggling multiple payout providers. Use a single integration to automate hours of operational overhead and reduce opportunities for human error.',
```
Comment (ralph-dev) [inline src/Containers/TableFeature/TableFeature.stories.tsx:46]: spelling

Comment (ralph-dev) [inline src/Containers/TableFeature/TableFeature.stories.tsx:46]: becuase this is a React UI Library, The component you create will need to be extremely Clear and Readable. <TableFeature by the props what goes where?

File: src/Containers/TableFeature/TableFeature.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    display: block;
+    width: 80%;
+    margin: 5rem auto;
+`;
+const HeaderWrapper = styled.div` 
+    display: flex; 
+    flex-direction: column;
+`; 
+const Heading = styled.div`
+    font-weight: 600;
```
Comment (ralph-dev) [inline src/Containers/TableFeature/TableFeature.tsx:18]: const BaseText = styled.div`
font-weight: 600;
`;

const Heading = styled(BaseText)`
marginBottm: 1px
`;

**Pull request closed without merging**